otx/algorithms/detection/configs/instance_segmentation/resnet50_maskrcnn/deployment.py

```diff
@@ -6,10 +6,10 @@
 
 ir_config = dict(
     output_names=["boxes", "labels", "masks"],
-    input_shape=(1344, 800),
+    # input_shape=(1344, 800),
```

Comment:
Why should we comment this?

Reply:
I'm not sure. IMO, It will be automatically patched. So, commenting on this line could be helpful to understand these values will not be used.

Reply:
https://github.com/openvinotoolkit/training_extensions/pull/2172/commits/9694b18b3e292379f8d278d0a4cf517d26d562db